### PR TITLE
refactor configure

### PR DESCRIPTION
**Pull request by ZhengzheYang**
This PR refactors the configure steps to take place in install subcommand and in each individual components.

### CODE REVIEW

File: asmcli/commands/install.sh [hunk 1]
```diff
@@ -200,3 +177,49 @@ outro() {
...
+    local CA_CHAIN; CA_CHAIN="$(context_get-option "CA_CHAIN")"
+
+    info "Installing certificates into the cluster..."
+    kubectl create secret generic cacerts -n istio-system \
+      --from-file="${CA_CERT}" \
+      --from-file="${CA_KEY}" \
+      --from-file="${CA_ROOT}" \
+      --from-file="${CA_CHAIN}"
+  fi
+}
```
Comment (zerobfd) [inline asmcli/commands/install.sh:211]: Everything looks good except this is Citadel specific, so it should co into a similar file as meshca.sh

**Review by zerobfd — APPROVED** [review]

File: asmcli/commands/install.sh (hunk 1 above)
Comment (ZhengzheYang) [reply asmcli/commands/install.sh]: Moved to an install function in citadel.sh

**Pull request merged**